SECURITY.md

```diff
@@ -4,11 +4,15 @@
 
 The versions of the project that are currently supported with security updates.
 
-| Version | Supported          |
-| ------: | :----------------- |
-|     6.x | :white_check_mark: |
-|     5.x | :white_check_mark: |
-|   < 5.0 | :x:                |
+| MUI X version | Release    | Supported                            |
```

Comment:
We should probably update https://www.notion.so/mui-org/MUI-X-Major-version-release-process-a27f296dfb9e4b33bdb767f3d6e8018c to add the changes that should be done to those files

Reply:
Added https://www.notion.so/mui-org/MUI-X-Major-version-release-process-a27f296dfb9e4b33bdb767f3d6e8018c?pvs=4#e1962b1f600a4cb6bdf613ca547a2688

Reply:
Great, I love to see the docs updated 👌